src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -953,27 +919,37 @@ std::string UsgsAstroFrameSensorModel::constructStateFromIsd(const std::string&
       state["m_transY"][2] =  state["m_iTransS"][2].get<double>() / determinant;
       state["m_transY"][0] = -(state["m_transY"][1].get<double>() * state["m_iTransL"][0].get<double>() +
                                state["m_transY"][2].get<double>() * state["m_iTransS"][0].get<double>());
+    }
+    catch (...) {
+      parsingWarnings->push_back(
+        csm::Warning(
+          csm::Warning::DATA_NOT_AVAILABLE,
+          "Could not compute detector pixel to focal plane coordinate transformation.",
+          "UsgsAstroFrameSensorModel::constructStateFromIsd()"));
+    }
 
-      std::cerr << "Focal To Pixel Transformation Parsed!" << std::endl;
+    std::cerr << "Focal To Pixel Transformation Parsed!" << std::endl;
```

Comment:
This comment is really for all of the `std::cerr` entries:

What are people's thoughts on spdlog? Header only, available on conda-forge, looks to implement many of the nice to have from python's logging module. 

https://github.com/gabime/spdlog
https://anaconda.org/search?q=spdlog

Reply:
These are hold overs from trying to get GXP working. I'd be up for just removing them.

Reply:
:+1: Let's just pull these then.